django_emarsys/service.py

```diff
@@ -0,0 +1,25 @@
+from django_emarsys import api
```

Comment:
The model is named after the concept, and this file contains the primary API for the concept. :)